src/_gettsim/combine_functions_in_tree.py

```diff
@@ -489,6 +492,106 @@ def aggregate_by_group_func(source_col, group_id):
     )
 
 
+def get_groupby_id(
```

Comment:
I do not quite understand why we need a different mechanism here than for policy functions, related to your commit message *("group-id-to-source-col match. Need to think about how to handle group IDs from input data")*. 

Could you elaborate? I think writing it down might help.

Reply:
Ah, I think I am slowly getting it. Yes, just hard-code.